.github/workflows/stream-connectors-dependencies.yml

```diff
@@ -79,19 +79,40 @@ jobs:
         working-directory: dependencies/${{ matrix.lib }}
         shell: bash
 
+      - if: ${{ matrix.lib == 'lua-curl' }}
+        uses: actions/checkout@v3
+        with:
+          repository: Lua-cURL/Lua-cURLv3
+          path: lua-curl-src
+          ref: v0.3.13
+
       - if: ${{ matrix.lib == 'lua-curl' }}
         run: |
-          luarocks install lua-curl
-          luarocks show lua-curl
-          luarocks show lua-curl | grep "Lua-cURL\s[0-9]" | cut -d' ' -f2 | perl -nle 'm/(\d+\.\d+(\.\d+)?)/; print $1' >> version.txt
-          cat version.txt
+          dnf install -y openssl-devel openssl libcurl-devel python3-pip perl-IPC-Cmd perl-Digest-SHA perl-Thread-Queue perl-IO-Socket-SSL
+
+          cd lua-curl-src
+
+          pip3 install conan
+          conan profile detect
+          conan install --requires=libcurl/8.1.2 --build=missing -g CMakeToolchain --deployer=full_deploy
+          conan install --requires=openssl/3.1.1 --build=missing -g CMakeToolchain --deployer=full_deploy
+          conan install --requires=zlib/1.2.13 --build=missing -g CMakeToolchain --deployer=full_deploy
+
+          sed -i "s#^CURL_LIBS.*#CURL_LIBS=-Lfull_deploy/host/libcurl/8.1.2/Release/x86_64/lib -l:libcurl.a -Lfull_deploy/host/openssl/3.1.1/Release/x86_64/lib -l:libssl.a -l:libcrypto.a -L full_deploy/host/zlib/1.2.13/Release/x86_64/lib -l:libz.a#" Makefile
+          make
+
+          cd ..
+
+          mkdir -p dependencies/${{ matrix.lib }}/${{ matrix.lib }}/
```

Comment:
why create twice "matrix.lib" folder ?

Reply:
this was already the structure
first level is the dependency directory in the git
second level is the directory of packaged files